src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable no-use-before-define */
-import FastImage from 'react-native-fast-image';
 import React from 'react';
 import {View} from 'react-native';
+import FastImage from 'react-native-fast-image';
 
-import CommunityIcon from '../../../assets/hashtag.png';
-import GroupIcon from '../../../assets/group-icon.png';
 import {
   ChannelImageBadgeProps,
   ChannelImageMainProps
 } from '../../../../types/component/ChatList/ChannelImage.types';
+import GroupIcon from '../../../assets/group-icon.png';
+import CommunityIcon from '../../../assets/hashtag.png';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered. This change doesn't affect the functionality of the code, but it's generally a good practice to keep imports organized. For instance, you could group and order them by library imports, absolute imports, and relative imports.
```diff
- import React from 'react';
- import {View} from 'react-native';
- import FastImage from 'react-native-fast-image';
- import {
-   ChannelImageBadgeProps,
-   ChannelImageMainProps
- } from '../../../../types/component/ChatList/ChannelImage.types';
- import GroupIcon from '../../../assets/group-icon.png';
- import CommunityIcon from '../../../assets/hashtag.png';
+ /* eslint-disable no-use-before-define */
+ import React from 'react';
+ import {View} from 'react-native';
+ import FastImage from 'react-native-fast-image';
+ import {
+   ChannelImageBadgeProps,
+   ChannelImageMainProps
+ } from '../../../../types/component/ChatList/ChannelImage.types';
+ import GroupIcon from '../../../assets/group-icon.png';
+ import CommunityIcon from '../../../assets/hashtag.png';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->